### DISCUSSION TITLE
dofferent colors for working area and staging are changes

### DISCUSSION THREAD

**Discussion by jobinjosem** [JanDeDobbeleer/oh-my-posh#3568] [General]
I was wondering if it is possible to configure different foreground colors for the changes in working area and staging area.
something like 

 "foreground_templates": [
            "{{ if .Working.Changed}}#FF9248{{ end }}",
            "{{ if .Staging.Changed}}#ff4500{{ end }}"
          ],
![image](https://user-images.githubusercontent.com/81211186/224178481-c9d3c0c6-8ed5-43e8-9cbf-46edc2218deb.png)
An example is attached

**Comment by JanDeDobbeleer** [comment]
@jobinjosem you need to use inline color overrides in the `template` and not the `foreground_templates`:

```json
"template": " <#8A8B07>[</>{{ .HEAD }}{{if .BranchStatus }} {{ .BranchStatus }}{{ end }}{{ if .Working.Changed }} <#FF9248>{{ .Working.String }}</>{{ end }}{{ if and (.Working.Changed) (.Staging.Changed) }} <#8A8B07>|</>{{ end }}{{ if .Staging.Changed }} <#FF9248>{{ .Staging.String }}</>{{ end }}<#8A8B07>]</>> ",

```

**Reply by jobinjosem** [reply]
Thanks 👍
